website/versioned_sidebars/version-2.9-sidebars.json

```diff
@@ -4,14 +4,8 @@
       "type": "category",
       "label": "API Client",
       "items": [
-        "index",
-        "features",
-        "usage_concepts"
+        "index"
       ]
-    },
-    {
-      "type": "doc",
-      "id": "changelog"
```

Comment:
why removing changelog from the sidebar?

Reply:
It's still in the navigation, we found it enough as it's not connected to the content.

![Snímek obrazovky 2024-05-03 v 10 05 15](https://github.com/apify/apify-client-js/assets/45016873/51c70f42-f411-43cf-9bac-b90b9b8a358a)